tools/hilbert.py

```diff
@@ -675,6 +675,41 @@ def cmd_app_change(parser, context, args):
 
     return args
 
+@subcmd('cfg_query', help="query configuration properties")
+def cmd_cfg_query(parser, context, args):
+    action = 'cfg_query'
+    log.debug("Running 'cmd_{}'".format(action))
+
+    group = parser.add_mutually_exclusive_group()
+
+    group.add_argument('--configfile', required=False,
+                       help="specify input .YAML file (default: 'Hilbert.yml')")
+    group.add_argument('--configdump', required=False,
+                       help="specify input dump file")
+
+    parser.add_argument('QueryString', help="specify the query")
+
+    args = parser.parse_args(args)
+
+    log.debug("Querying '{}' from the configuration...".format(args.QueryString))
+
+    obj = None
+    try:
+        obj = cmd_list(parser, context, args, args.QueryString)
+    except:
+        log.exception("Sorry could not get the result of query '{}' from the input file!".format(args.QueryString))
+        sys.exit(1)
+
+    assert obj is not None
+
+    if isinstance(obj, BaseValidator):
+        yaml_dump(obj.data_dump(),sys.stdout)
```

Comment:
This prints a separate line containing `...` for many (most?) single-line dumps. I don't know if this is a bug or intended behaviour. At least it's not very useful nor intuitive if you want to use the result in subsequent scripts.

Reply:
should not happen. may be a bug. small sample code for unit-test plz?